packages/core/bootstrap/src/lib/cache-warmer/epics.ts

```diff
@@ -174,18 +174,44 @@ export const warmupRequestHandler: Epic<AnyAction, AnyAction, any> = (action$, s
     filter(warmupRequested.match),
     // fetch our required state to make a request to warm up an adapter
     withLatestFrom(state$),
-    map(([action, state]) => ({
-      requestData: state.cacheWarmer.subscriptions[action.payload.key],
-      key: action.payload.key,
-    })),
+    map(([action, state]) => {
+      const requestData = state.cacheWarmer.subscriptions[action.payload.key]
+      let batchablePropertyPath
```

Comment:
Actually it may just be easier to store `batchablePropertyPath` on the parent/batch warmer as well and skip doing this completely.

Reply:
We don't have `batchablePropertyPath` when we save the parent in Redux though.